js/main.js

```diff
@@ -27,4 +27,96 @@ function handleSubmit(event) {
   data.entries.unshift(inputValues);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+  renderEntry(0);
+  $ul.prepend(renderEntry(0));
```

Comment:
You're calling `renderEntry` twice, it should only be called once, like on line 31